src/components/card/Description.vue

```diff
@@ -263,6 +290,40 @@ export default {
 				await this.saveDescription()
 			}, 2500)
 		},
+		async addLastAttachmmentOnCursor() {
+			const attachementLength = this.$store.getters.attachmentsByCard(this.card.id).length
+			if (attachementLength === 0) {
+				return
+			}
+			this.addAttachment(this.$store.getters.attachmentsByCard(this.card.id)[attachementLength - 1])
+		},
+		onPaste() {
+			if (!this.descriptionEditing) {
+				return
+			}
+
+			const clipboardData = event.clipboardData
+			let files = []
+			if (clipboardData.files && clipboardData.files.length > 0) {
+				files = clipboardData.files
+			} else if (clipboardData.items && clipboardData.items.length > 0) {
+				for (let i = 0; i < clipboardData.items.length; i++) {
+					if (clipboardData.items[i].kind === 'file') {
+						files.push(clipboardData.items[i].getAsFile())
+					}
+				}
+			}
+
+			if (files.length === 0) {
+				return
+			}
+
+			this.loading = true
+			event.preventDefault()
```

Comment:
This doesn't seem to work in Firefox as I still receive a path to my file in the text of the document:

![image](https://user-images.githubusercontent.com/986878/206710830-09e2060a-3520-4812-ac4d-76603ecc29a6.png)